recipes/reikna/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "reikna" %}
+{% set version = "0.6.8" %}
+{% set sha256 = "34d92786237bef9ab5d37d78f01c155d0dcd1fc24df7782af9498a9f1786890c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - mako >=0.8.0
+    - numpy >=1.6.0
+    - funcsigs >=0.3
+    # Optional
+    # - pycuda
+    # - pyopencl
```

Comment:
Either `pycuda` or `pyopencl` can be used. Currently `conda-forge` has `pyopencl`. However `pycuda` doesn't appear in `conda-forge` or `defaults`. Packaging `pycuda` would require CUDA headers and libraries.